utilities/utils.go

```diff
@@ -0,0 +1,15 @@
+package utilities
```

Comment:
I thought this was supposed to be an before hook function

Reply:
Yes, I could take it into the before-hook function but then I'm looking at a possibility of having to use it in another context. Although I can't think of any other use case now, I would want to believe it's safe to just keep it in the utilities altogether. If you feel strongly otherwise though, I can make the change and we then refactor if any use case arrives in the future.

Reply:
it's ok